app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -561,13 +587,18 @@ impl Container {
         let action_bar = &model.action_bar.frp;
         let registry = &model.registry;
         let selection = Animation::new(network);
+        let width_anim = Animation::new(network);
+
+        let on_down = model.view.resize_grip.on_event_capturing::<mouse::Down>();
```

Comment:
Why do we need to do this on capturing phase? Shouldn't the grip be just over elements?

Also, the var name does not say it's _resizing_ mouse down, so looking 30 lines below we may miss this information.